pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_child_contact_1wk_reminder_es.html

```diff
@@ -98,6 +98,8 @@
   </tr>
   <tr style="width: 540px;">
     <td style="padding: 20px 0 80px 0;">
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Si no nos proporciona el correo electrónico de su hijo (a) o proporciona el correo electrónico de su hijo, pero su hijo (a) no completa los pasos requeridos para la renovación del consentimiento, concluiremos que tenemos el permiso implícito de su hijo (a) para continuar en el proyecto de investigación. Por favor informe a su hijo (a) que su participación continuará después de cumplir 18 años a menos que se comunique con nosotros ya sea a través del proceso de renovación de consentimiento o en la información de contacto a continuación para informarnos de su deseo de retirarse del estudio.</p>
```

Comment:
per source should be:

> o si proporciona el correo 

just need to add a 'si' and '(anteriormente Twitter)' and then this one's good